src/components/Header.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.